Templates/CSharp/Model/ComplexType.cs.tt

```diff
@@ -26,14 +26,19 @@ if (!complex.IsAbstract && complex.Derived != null && (complex.Base == null || c
     attributeStringBuilder.Append("[JsonConverter(typeof(DerivedTypeConverter))]");
 }
 
+var namespaceName = complex.Namespace.GetNamespaceName();
+
 #>
 
-namespace <#=complex.Namespace.GetNamespaceName()#>
+namespace <#=namespaceName#>
 {
     using System;
     using System.Collections.Generic;
     using System.IO;
     using System.Runtime.Serialization;
+<# if (!this.IsGraphNamespace(namespaceName)) { #>
+    using Microsoft.Graph;
```

Comment:
Rather than doing something like this, would it not work to just fully qualify types where this matters?